test/sanity/sanity_test.go

```diff
@@ -36,10 +37,32 @@ func TestSanity(t *testing.T) {
 		driver.Run(context.Background(), "secrets-store.csi.k8s.io", "somenodeid", endpoint, providerVolumePath, "provider1=0.0.2,provider2=0.0.4", "", nil)
 	}()
 
+	tmpPath := path.Join(os.TempDir(), "csi")
 	config := &sanity.Config{
-		TargetPath:  mountPath,
-		StagingPath: stagePath,
+		TargetPath:  os.TempDir() + "/csi-target",
+		StagingPath: os.TempDir() + "/csi-staging",
 		Address:     endpoint,
+		CreateTargetDir: func(targetPath string) (string, error) {
+			targetPath = path.Join(tmpPath, targetPath)
+			return targetPath, createTargetDir(targetPath)
+		},
+		RemoveTargetPath: func(targetPath string) error {
+			return os.RemoveAll(targetPath)
```

Comment:
Should this be `os.RemoveAll(tmpPath)`?

Reply:
This is for each individual sanity test, so we only need to cleanup the target path.